src/helpers/__tests__/dateUtils.test.js

```diff
@@ -29,4 +29,18 @@ describe('date utils', () => {
       expect(wrapper.text()).toContain('Fri, May 4');
     });
   });
+
+  describe('formatTimeDuration', () => {
+    it('should format duration', () => {
+      expect(dateUtils.formatTimeDuration(90)).toBe('1h 30min');
+      expect(dateUtils.formatTimeDuration(32)).toBe('32min');
+      expect(dateUtils.formatTimeDuration(191)).toBe('3h 11min');
+    });
+  });
+  describe('formatHour', () => {
+    it('should format duration', () => {
+      const myDate = '2018-05-14T13:30:16.905Z';
+      expect(dateUtils.formatHour(myDate)).toBe('3:30 PM');
```

Comment:
this isn't correct, it should display `1:30 PM`

CI fails correcly
toLocaleString needs to take `timeZone: 'UTC'` as part of the options

Reply:
Yep, you're right. I was making sure all graphql times are UTC-timezoned, and that is the case. Fixing this now, thx.